hazelcast/src/main/java/com/hazelcast/util/SortingUtil.java

```diff
@@ -128,9 +128,9 @@ public int compare(Map.Entry entry1, Map.Entry entry2) {
         };
     }
 
-    public static Comparator<Map.Entry> newComparator(final PagingPredicate pagingPredicate) {
```

Comment:
This class is my main point of worry. Officially hazelcast.util is private, so there should not be any problems if an external user would be running in a potential incompatibility problem..